internal/configs/configschema/filter.go

```diff
@@ -26,6 +26,25 @@ var (
 	FilterDeprecatedBlock = func(path cty.Path, block *NestedBlock) bool {
 		return block.Deprecated
 	}
+
+	// FilterAttrByName returns a negative filter, meaning it excludes attributes with the given name.
+	FilterAttrByName = func(name string) FilterT[*Attribute] {
```

Comment:
If we're just going to have data be a computed dynamic type, we can drop this filtering idea for now.

Reply:
We still need to remove it from the config, but yeah there is an[ existing filter](https://github.com/hashicorp/terraform/blob/86e42760e2bdbe59ca477d6a7be97bf8915c4a14/internal/configs/configschema/filter.go#L11-L13) for removing read-only attributes